config/core/test-configs-chromeos.yaml

```diff
@@ -148,14 +148,29 @@ device_types:
         image: 'https://storage.chromeos.kernelci.org/images/rootfs/chromeos/chromiumos-hatch/20220621.0/amd64/bzImage'
         modules: 'https://storage.chromeos.kernelci.org/images/rootfs/chromeos/chromiumos-hatch/20220621.0/amd64/modules.tar.xz'
 
-  # Placeholder
-  # hp-11A-G6-EE-grunt_chromeos:
-  #   base_name: hp-11A-G6-EE-grunt
-  #   mach: x86
-  #   arch: x86_64
-  #   boot_method: depthcharge
-  #   params:
-  #     block_device: detect
+  hp-11A-G6-EE-grunt_chromeos:
+    base_name: hp-11A-G6-EE-grunt
+    mach: x86
+    arch: x86_64
+    boot_method: depthcharge
+    params: &grunt-params
+      cros_flash_nfs:
+        base_url: 'https://storage.chromeos.kernelci.org/images/rootfs/chromeos/chromiumos-grunt/20220711.0/amd64/'
```

Comment:
This one need to be set:
        base_url: 'https://storage.kernelci.org/images/rootfs/debian/bullseye-cros-flash/20220718.0/amd64/'
similar to octopus and hatch

Reply:
thanks @nuclearcat. Updated.